ui/app/styles/components/codemirror.scss

```diff
@@ -156,3 +156,29 @@ header.run-job-header {
   display: flex;
   justify-content: space-between;
 }
+
+.select-mode {
```

Comment:
Request: have this be more specific, either by class name or by nesting. As it stands, I could see someone creating another component with a class of `select-mode` in the future and not being sure why there's suddenly a fresh looking grid applied.

Reply:
The #2 problem of programming is naming things. I can have ChatGPT name this or take a name from you! Is that unreasonable?

Reply:
I didn't actually write this code FYI. I [stole](https://github.com/hashicorp/nomad/pull/16134/files#diff-2a532068e46093a6e751d50034579f500880b133d580718bab64a1ec5096630fR2-R26) it from your work.